addon/components/swiper-container.js

```diff
@@ -206,19 +204,36 @@ export default Component.extend({
    * @param {Object} swiper - Swiper instance
    */
   _slideChanged(swiper) {
-    let index = this.get('loop')
-      ? $(swiper.slides)
-        .filter('.swiper-slide-active')
-        .attr('data-swiper-slide-index')
-      : swiper.realIndex;
+    let index;
+
+    if (this.get('loop')) {
+      Object.keys(swiper.slides).forEach((key) => {
+        if (swiper.slides[key].classList && swiper.slides[key].classList.contains('swiper-slide-active')) {
+          index = swiper.slides[key].getAttribute('data-swiper-slide-index');
+        }
+      });
+    } else {
+      index = swiper.realIndex;
+    }
     this.set('_currentSlideInternal', index);
     this.set('currentSlide', index);
     this.get('onChange')(swiper.slides[swiper.realIndex]);
   },
-
+  /**
+   * Get all previous elements
+   * @private
+   * @param {Object} DOMElement - A Dom Element
+   * @return {Array}
+  */
+  prevAll(element) {
```

Comment:
We may want to indicate that this is a private method with: `_prevAll`?

Reply:
I would make the `@return {Array}` into `@return {HTMLElement[]}` as it's a bit more communicative.

Reply:
Yeah, will do